README.md

```diff
@@ -1,3 +1,8 @@
+<!--[![Build master branch](https://github.com/CWTSLeiden/networkanalysis/workflows/Build%20master%20branch/badge.svg?branch=master)](https://github.com/CWTSLeiden/networkanalysis/actions)//-->
```

Comment:
We can (should) uncomment this before merging.

Reply:
Good point!